ci/destroy-infrastructure.yaml

```diff
@@ -0,0 +1,20 @@
+platform: linux
+inputs:
+  - name: eq-terraform-load-generator
+params:
+  GOOGLE_APPLICATION_CREDENTIALS: /root/gcloud-service-key.json
```

Comment:
I think the destroy needs the folder id and billing account as my destroy is hanging with the output:
```
var.gcp_billing_account
  The billing account the project will use, defaults to the Census CTS account`.
```